backend/tournesol/views/polls.py

```diff
@@ -22,6 +22,24 @@
 
 logger = logging.getLogger(__name__)
 
+WEIGHTS_PARAMETER = OpenApiParameter(
+    "weights",
+    OpenApiTypes.OBJECT,
+    style="deepObject",
+    description="Weights for criteria in this poll."
+                " The default weight is 10 for each criteria.",
```

Comment:
Isn't the default weight 50 ?

Reply:
It looks like it changed recently. Maybe to allow for more arbitrary weights than just percentages.